warehouse/integrations/bigquery/bigquery.go

```diff
@@ -264,6 +271,18 @@ func (bq *BigQuery) deduplicationQuery(tableName string, columnMap model.TableSc
 		partitionColumn, partitionType = bq.partitionColumn(), bq.partitionType()
 	)
 
+	// if partitioningInWarehouse is not nil, then we use the partition column and type from the warehouse
+	if partitioningInWarehouse != nil {
+		if partitionColumn != partitioningInWarehouse.Field {
+			bq.logger.Warnn("partition column mismatch in config and warehouse",
```

Comment:
Do you think we can make this a debug? It will become noise because it will come every time for the customer if they have modified something.

Reply:
We can make it debug. We are logging sql queries, so we will get to know what partition column is being used.